index.bs

```diff
@@ -4377,6 +4378,38 @@ navigated</dfn> steps given |context| and |navigation status|:
 
 </div>
 
+#### The browsingContext.historyUpdated Event #### {#event-browsingContext-historyUpdated}
+
+<dl>
+   <dt>Event Type</dt>
+   <dd>
+      <pre class="cddl local-cddl">
+        browsingContext.HistoryUpdated = (
+         method: "browsingContext.historyUpdated",
+         params: {
+            url: text,
+         }
```

Comment:
We should at least have the `context` id where the history update happened? I know this is not a navigation but a `NavigationInfo` with navigation=null (which is supported) would probably be a good fit?

Reply:
Updated. History updates are handled differently than navigations so I do not think NavigationInfo would be a good fit.